src/lib/utils/hooks/SynapseAPI/entity/useEntity.ts

```diff
@@ -33,7 +36,7 @@ export function useGetEntity<T extends Entity>(
 ) {
   const { accessToken } = useSynapseContext()
   return useQuery<T, SynapseClientError>(
-    ['entity', entityId, 'entity', versionNumber],
+    entityQueryKeys.version(entityId, versionNumber),
```

Comment:
All Entity-related queries should reference the enumerated query keys, rather than specifying their own.